versioned_docs/version-v1.3/troubleshooting/host.md

```diff
@@ -0,0 +1,139 @@
+time="2024-08-05T19:03:07Z" level=info msg="error when evicting pods/\"instance-manager-68cd2514dd3f6d59b95cbd865d5b08f7\" -n \"longhorn-system\" (will retry after 5s): Cannot evict pod as it would violate the pod's disruption budget."
+
+time="2024-08-05T19:03:12Z" level=info msg="evicting pod longhorn-system/instance-manager-68cd2514dd3f6d59b95cbd865d5b08f7"
+time="2024-08-05T19:03:12Z" level=info msg="error when evicting pods/\"instance-manager-68cd2514dd3f6d59b95cbd865d5b08f7\" -n \"longhorn-system\" (will retry after 5s): Cannot evict pod as it would violate the pod's disruption budget."
+```
+
+The Longhorn Instance Manager uses a PodDisruptionBudget (PDB) to protect itself from accidental eviction, which results in loss of volume data. When the Maintenance Mode error occurs, it indicates that the `instance-manager` pod is still serving volumes or replicas.
+
+The following sections describe the known causes and their corresponding workarounds.
+
+### Manually Attached Volumes
+
+A volume that is attached to a node using the [embedded Longhorn UI](./harvester.md#access-embedded-rancher-and-longhorn-dashboards) can cause the error. This is because the object is attached to a node name instead of the pod name.
+
+You can check it from the [Embedded Longhorn UI](./harvester.md#access-embedded-rancher-and-longhorn-dashboards).
+
+![attached-volume.png](/img/v1.3/troubleshooting/attached-volume.png)
+
+The manually attached object is attached to a node name instead of the pod name.
+
+You can also use the CLI to retrieve the details of the CRD object `VolumeAttachment`.
+
+Example of a volume that was attached using the Longhorn UI:
+
+```
+- apiVersion: longhorn.io/v1beta2
+  kind: VolumeAttachment
+...
+  spec:
+    attachmentTickets:
+      longhorn-ui:
+        id: longhorn-ui
+        nodeID: node-name
+...
+    volume: pvc-9b35136c-f59e-414b-aa55-b84b9b21ff89
+```
+
+Example of a volume that was attached using the Longhorn CSI driver:
+
+```
+- apiVersion: longhorn.io/v1beta2
+  kind: VolumeAttachment
+  spec:
+    attachmentTickets:
+      csi-b5097155cddde50b4683b0e659923e379cbfc3873b5b2ee776deb3874102e9bf:
+        id: csi-b5097155cddde50b4683b0e659923e379cbfc3873b5b2ee776deb3874102e9bf
+        nodeID: node-name
+...
+    volume: pvc-3c6403cd-f1cd-4b84-9b46-162f746b9667
+```
+
+:::note
+
+Manually attaching a volume to the node is not recommended.
+
+:::
+
+#### Workaround 1: Set `Detach Manually Attached Volumes When Cordoned` to `True`
+
+The Longhorn setting [Detach Manually Attached Volumes When Cordoned](https://longhorn.io/docs/1.6.0/references/settings/#detach-manually-attached-volumes-when-cordoned) blocks node draining when there are volumes manually attached to the node.
+
+The default value of this setting depends on the embedded Longhorn version:
+
+| Harvester version | Embedded Longhorn version | Default value |
+| --- | --- | --- |
+| v1.3.1 | v1.6.0 | `true` |
+| v1.4.0 | v1.7.0 | `false` |
```

Comment:
Does it make sense to mention 1.4.0 in the versioned documentation of 1.3.x? It makes sense for `1.4 (dev)`, but `1.3 (stable)`?

Reply:
I have no strong option to keep/remove it. This is a later update to v1.3 document, now we have a detailed plan about v1.4, add the section to v1.4 seems no harm.

But this more depends on the suggestion from @jillian-maroket . thanks.